mlflow/bedrock/_autolog.py

```diff
@@ -0,0 +1,119 @@
+import io
+import json
+from typing import Any, Callable, Union
+
+from botocore.client import BaseClient
+from botocore.eventstream import EventStream
+
+import mlflow
+from mlflow.bedrock import FLAVOR_NAME
+from mlflow.entities import SpanType
+from mlflow.utils.autologging_utils import safe_patch
+from mlflow.utils.autologging_utils.config import AutoLoggingConfig
+
+_BEDROCK_RUNTIME_SERVICE_NAME = "bedrock-runtime"
```

Comment:
Where did you get these constants? Would be helpful if you could add a permanent link to the references.

Reply:
Right, basically boto3 provides two types of clients for bedrock:

1. `Bedrock` - for CRUD operations on the bedrock resources themselves. E.g., creating an LLM endpoint.
2. `BedrockRuntime` - for the actual usage of those bedrock resources. E.g., querying an LLM endpoint.

For our case, we want to trace the latter.